modules/ndu/src/backend/index.ts

```diff
@@ -20,7 +21,22 @@ const onServerStarted = async (bp: typeof sdk) => {
 }
 
 const onServerReady = async (bp: typeof sdk) => {
-  await api(bp, bots)
+  const router = bp.http.createRouterForBot('ndu')
```

Comment:
why not place the router in `api.ts` like in every other module?

Reply:
Because there's almost nothing, plus the conditions route will be removed in another pr.

Not sure the events route will stay there either